src/harness/BenchmarkDotNet.Extensions/ValuesGenerator.cs

```diff
@@ -71,6 +71,32 @@ public static T[] Array<T>(int count)
             return result;
         }
 
+        public static readonly byte[] s_encodingMap = {
+            65, 66, 67, 68, 69, 70, 71, 72,         //A..H
+            73, 74, 75, 76, 77, 78, 79, 80,         //I..P
+            81, 82, 83, 84, 85, 86, 87, 88,         //Q..X
+            89, 90, 97, 98, 99, 100, 101, 102,      //Y..Z, a..f
+            103, 104, 105, 106, 107, 108, 109, 110, //g..n
+            111, 112, 113, 114, 115, 116, 117, 118, //o..v
+            119, 120, 121, 122, 48, 49, 50, 51,     //w..z, 0..3
+            52, 53, 54, 55, 56, 57, 43, 47          //4..9, +, /
+        };
+
+        public static byte[] ArrayBase64EncodingBytes(int count)
+        {
+            var result = new byte[count];
+
+            var random = new Random(Seed);
+
+            for (int i = 0; i < result.Length; i++)
+            {
+                int index = (byte)random.Next(0, s_encodingMap.Length - 1);    // Do not pick '='
```

Comment:
Not sure I understand the comment `// Do not pick '='`. I don't see it in the `s_encodingMap`

Reply:
I just read it as "= is often seen in base64 output, but it's just a delimiter, don't include it in the bytes"

Reply:
> the 2nd parameter is an exclusive upper bound

That's what I meant.

Reply:
oh - ha - got it, we have the same observation

Reply:
Yeah, agreed that looks wrong.
I just copied the table and function from the library tests in dotnet:

https://github.com/dotnet/runtime/blob/41419131095d36fb5b811600ad0dab3b0d804269/src/libraries/System.Memory/tests/Base64/Base64TestHelper.cs#L16

https://github.com/dotnet/runtime/blob/41419131095d36fb5b811600ad0dab3b0d804269/src/libraries/System.Memory/tests/Base64/Base64TestHelper.cs#L75

I wonder if an older version of the code had an = sign on the end of the array.

Either way, I'll update this to:
int index = (byte)random.Next(0, s_encodingMap.Length);

And I guess I could fix it in dotnet by squeezing it into my intrinsic patch.

Reply:
yeah, I was going to ask whether you'd mind fixing it there too. might be easiest to throw it up in its own PR. thanks!

Reply:
It seemed a little small to put in it's own PR, but that's fine, I'll do that

Reply:
Removed from this PR